mep/people/views.py

```diff
@@ -29,7 +29,14 @@ class MembersList(ListView):
     def get_queryset(self):
         # limit to people with library accounts, do not include
         # item creators who are not library members
-        return super().get_queryset().exclude(account=None)
+        return super().get_queryset().exclude(account=None).exclude(mep_id='')
```

Comment:
The mep_id filter here kind of seems like cheating. I checked latest data and there are over 2,000 library members without an mepid, and excluding them from the list view seems problematic and potentially confusing for our testers. If you want to use the mepid for the current iteration, you need to add fallback logic - I guess db id? Alternatively we could just use dbid for now (no point writing extra code that we may not need) and clearly comment it as preliminary.